app/src/main/resources/io/apicurio/registry/storage/impl/sql/upgrades.3/postgresql.upgrade.ddl

```diff
@@ -0,0 +1,37 @@
+-- *********************************************************************
+-- DDL for the Apicurio Registry - Database: PostgreSQL
+-- Upgrades the DB schema from version 2 to version 3.
+-- *********************************************************************
+
+-- Update database version to 3
+UPDATE apicurio SET prop_value = 3 WHERE prop_name = 'db_version';
+
+-- new sequences table
+CREATE TABLE sequences(tenantid character varying(128) NOT NULL, name character varying(32) NOT NULL, nextval bigint NOT NULL);
```

Comment:
Should use consistent casing on the column names even though it doesn't matter in the actual DB layer.  So e.g. use `tenantId` instead of `tenantid`.